components/tooling/glean-gradle-plugin/README.md

```diff
@@ -0,0 +1,3 @@
+This provides a Gradle plugin that generates Glean metrics from metrics.yaml and pings.yaml files.
```

Comment:
Can you carry over the phrasing from [here](https://github.com/mozilla/glean/blob/master/gradle-plugin/README.md)?